src/SingleOrderForm.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { updateOrder, getOrderById, getSingleOrderItemsUsers } from './store/order';
+import { fetchLineItemsByOrder } from './store/lineitem';
+import Errors from './Errors';
+
+class SingleOrder extends Component {
+    constructor(){
+        super();
+        this.state = {
+            status: '',
+            errors: []
+        };
+    }
+    
+    componentDidMount() {
+        this.props.getSingleOrderItemsUsers(this.props.user.id, this.props.match.params.orderId)
+            .then(response => response.data)
+            .then(() => this.setState({ status: this.props.order.status, errors: [] }))
+            .catch(e => console.log(e.message))
+    }
+
+    componentDidUpdate(prevProps) {
+        if(JSON.stringify(prevProps) !== JSON.stringify(this.props)){
+            this.props.getSingleOrderItemsUsers(this.props.user.id, this.props.match.params.orderId)
```

Comment:
I am seeing a 500 error for get /api/orders/user/undefined, when click on save or submit on single order form. Maybe I need to sync with the latest dev code